examples/stereo_panner.rs

```diff
@@ -0,0 +1,62 @@
+extern crate servo_media;
+extern crate servo_media_auto;
+
```

Comment:
Name this file `stereo_panner.rs`. Also there will need to be an entry in `examples/Cargo.toml` so that it can be run

Reply:
whoops, the name is already correct. you'll still need the entry though